geopandas/tests/test_sindex.py

```diff
@@ -1,13 +1,15 @@
 import sys
 
-from shapely.geometry import Point, Polygon, box
+from shapely.geometry import Point, Polygon, box, GeometryCollection
 from numpy.testing import assert_array_equal
 
 import geopandas
+from geopandas.array import _shapely_to_geom as shapely_to_geom
```

Comment:
@jorisvandenbossche I'd like to get your opinion on this. Generally, I like to respect the use of `_`, but I also don't think it's ideal to re-implement this here just to avoid the import.

Reply:
I think that should be handled inside the tree `query` method (ensuring that an input of shapely geometry gets converted to a pygeos geometry). And then in the tests here you can just use shapely's box.

Reply:
> I assume we're okay not checking in bulk_query?

Yes, that should be OK.

Reply:
Cool, change implemented then.